hnix.cabal

```diff
@@ -95,6 +95,9 @@ library
     , exceptions
     , filepath
     , hashable
+    , http-types
```

Comment:
We don't edit the `hnix.cabal` directly, but edit `package.yaml` and run `hpack`. There's a test that should have failed for you because of this...